modelgym/xgboost_model.py

```diff
@@ -1,10 +1,18 @@
 import xgboost as xgb
 from hyperopt import hp
 
-from modelgym.model import Model
+from modelgym.model import Model, TASK_CLASSIFICATION
 
 
 class XGBModel(Model):
+    defspace = {
```

Comment:
the idea was to have single definition of space that can be used by both hyperopt and skopt... not double them